Code/Scripts/Escape/CreateSearchChopper.sqf

```diff
@@ -53,10 +53,21 @@ _homePos = _homePos vectorAdd [0,0,100];
 //_chopperspawn = [_homePos, random 360, _type, A3E_VAR_Side_Opfor] call BIS_fnc_spawnVehicle;
 _chopper = createVehicle [_type, _homePos, [], random 360, "FLY"];
 [_chopper] call a3e_fnc_onVehicleSpawn;
-createVehicleCrew _chopper;
+private _group = createVehicleCrew _chopper;
+
 {
 	_x call drn_fnc_Escape_OnSpawnGeneralSoldierUnit;
 } foreach crew _chopper;
+
+// populate FFV turrets
+private _classList = [a3e_arr_Escape_InfantryTypes_Ind, a3e_arr_Escape_InfantryTypes] select (side _group == A3E_VAR_Side_Opfor);
+{
+	if !(_chopper lockedTurret _x) then {
+		private _soldier = _group createUnit [selectRandom _classList, [0,0,0], [], 0, "NONE"];
+		_soldier moveInTurret [_chopper, _x];
+	};
+} forEach (allTurrets [_chopper, true] - allTurrets [_chopper, false]);
```

Comment:
Biki mentions that the commander turret is also a FFV turret, but crew is already spawned into commander seat by BIS_Fnc_SpawnVehicle. Have you checked that this case is accounted for? Otherweise an extra soldier might be spawned at [0,0,0] and will wander across the terrain.

Reply:
:+1: fixed